popup/popup.html

```diff
@@ -11,13 +11,34 @@
     <div id="check_options" class="popup-font popup-info">
       To get live stream notifications, enter your API key and turn on the Live
       Stream Notifications option in the Add-On Options page.
+      <div class="option-text">
```

Comment:
I feel that the presentation of the option item in the popup is not ideal. I'd prefer a modification of the above text:

> To use the live stream indicator and quick nav, enter your [API key](https://www.giantbomb.com/api/) below:

Follow this with the input with a placeholder of "API key".

Reply:
That sounds much better.  I'll add that in.